pkg/ui/workspaces/cluster-ui/src/statsTableUtil/statsTableUtil.tsx

```diff
@@ -290,6 +292,43 @@ export const statisticsTableTitles: StatisticTableTitleType = {
       </Tooltip>
     );
   },
+  rowsWritten: (statType: StatisticType) => {
+    let contentModifier = "";
+    const fingerprintModifier = "";
+    switch (statType) {
+      case "transaction":
+        contentModifier = contentModifiers.transaction;
+        break;
+      case "statement":
+        contentModifier = contentModifiers.statements;
+        break;
+    }
+
+    return (
+      <Tooltip
+        placement="bottom"
+        style="tableTitle"
+        content={
+          <>
+            <p>
+              {"Aggregation of all rows "}
+              <Anchor href={writtenToDisk} target="_blank">
+                written to disk
+              </Anchor>
+              {` across all operators for ${contentModifier} with this fingerprint${fingerprintModifier} within the last hour or specified `}
+              <Anchor href={statementsTimeInterval} target="_blank">
+                time interval
+              </Anchor>
+              . The gray bar indicates the mean number of rows written to disk.
```

Comment:
I noticed that other components split this text up into two paragraphs. However, it seems that there is no space after the period with that format, since the linter complains if you add a space.

![image](https://user-images.githubusercontent.com/29153209/134421899-83e5e5a7-04a7-4fc9-800e-95a4b781ab57.png)

Two solutions I've found around this are:
1. adding a &nbsp after the period
2. putting everything in one paragraph and adding a space after the period

Wondering if my current approach is fine, and if I should also apply this to the other tooltips? (or if it's not that important of an issue?)

Reply:
I'd prefer the `&nbsp;` approach with 2 paragraphs because it makes the description more clear.